src/gui/mzroll/background_peaks_update.cpp

```diff
@@ -587,11 +587,21 @@ void BackgroundPeakUpdate::align() {
                         // if jDoc is null that means the json returned from python is malformed
                         // in such a case our rts wont update with new values
                         jDoc = QJsonDocument::fromJson(processedDataFromPython);
+
+                        QString errorMessage=QString::number(processedDataFromPython.size());
+                        errorMessage="Received data: "+errorMessage+" bytes";
+
                         if(!jDoc.isNull()){
                                 parentObj = jDoc.object();
                         }
                         else{
-                                qDebug()<<"Error: alignment- data is not complete";
+                                if(processedDataFromPython.size()==0){
```

Comment:
The whole error checking should be performed separately,no ? We should first check the exit status from python and then perform these checks to form the error message. 
Thoughts?